configuration/analyzer_config.json

```diff
@@ -3247,6 +3247,20 @@
     },
     "params": {}
   },
+  "Virushee_File_Scan":{
+    "type": "file",
+    "python_module": "virushee.Virushee",
+    "description": "Scan a File using VirusHee API",
+    "disabled": false,
+    "external_service": true,
+    "leaks_info": true,
+    "config": {
+      "soft_time_limit": 180,
+      "queue": "long"
+    },
+    "secrets": {},
+    "params": {}
+  },
```

Comment:
see my comment here: https://github.com/intelowlproject/IntelOwl/issues/440#issuecomment-988828006

Reply:
Please check this in the recent commits and I'm quite unsure about it as I could not find any existing analyzer with this `force_scan` param so did it as per my understanding and ignore the logging statements I'll remove them when PR is ready for review.